packages/runtime/container-runtime/src/test/dataStoreContext.spec.ts

```diff
@@ -225,9 +225,9 @@ describe("Data Store Context Tests", () => {
 
             remotedDataStoreContext = new RemotedFluidDataStoreContext(
                 dataStoreId,
-                Promise.resolve(snapshotTree),
+                snapshotTree,
```

Comment:
sooo much promise wrapping can be removed, as these things are always synchronously avaialable, it's just the SHA that made them async